tests/platforms/kubernetes/appmanager_test.go

```diff
@@ -130,12 +138,12 @@ func TestWaitUntilDeploymentState(t *testing.T) {
 				assert.Equal(t, testNamespace, ns)
 
 				switch action.GetVerb() {
-				case "create":
+				case createVerb:
 					// return the same deployment object
 					createdDeploymentObj = action.(core.CreateAction).GetObject().(*appsv1.Deployment)
 					createdDeploymentObj.Status.Replicas = testApp.Replicas
 
-				case "get":
+				case getVerb:
 					// return notfound error when WaitUntilDeploymentState called get deployments 2 times
 					if getVerbCalled == 2 {
```

Comment:
Can you make 2 a constant. It seems to be a magic number you chose to error out.

Reply:
This is the unit-test for test appmanager. This magic number is used only in this unit-test. so I would keep it as a number literal. also I also leave the comment

Reply:
My thought to make it a constant was not to make it reusable. Even though it is only used here, I see the magic 2 in multiple places. Are they referring the same config? I mean, can one of the 2s change to 3 without breaking the expected behavior? Second, making it a constant improves readability (IMO).